src/common/sorted-table/index.js

```diff
@@ -343,18 +445,45 @@ export default class SortedTable extends Component {
 
               const { id = i } = item
 
-              return rowLink
+              const selectionColumn = groupedActions != null && <td>
```

Comment:
Why not `groupedActions !== undefined`?

Even better: you should handle the case where this array is empty.